src/abstractdataframe/show.jl

```diff
@@ -203,38 +198,38 @@ julia> df = DataFrame(A = 1:3, B = ["x", "yy", "z"]);
 
 julia> maxwidths = DataFrames.getmaxwidths(df, stdout, 1:1, 3:3, :Row)
 3-element Array{Int64,1}:
- 1
- 1
+ 5
+ 6
  3
 
-julia> DataFrames.getchunkbounds(maxwidths, true, displaysize()[2])
-2-element Array{Int64,1}:
- 0
- 2
+julia> DataFrames.getchunkedcols(maxwidths, true, displaysize()[2])
+1-element Array{Array{Any,1},1}:
+ [1, 2]
 ```
 """
-function getchunkbounds(maxwidths::Vector{Int},
-                        splitcols::Bool,
-                        availablewidth::Int) # -> Vector{Int}
+function getchunkedcols(maxwidths::Vector{Int},
+                         splitcols::Bool,
+                         availablewidth::Int) # -> Vector{Int}
```

Comment:
wrong return value specification. I think you can just drop it, as you document it in the docstring.

Reply:
Good catch - I'm still getting used to this notation. I'm curious why this is used instead of a return type assertion. Is it just that this code predates that language feature?

```julia
function getchunkedcols(x, y, z)::Vector{Vector{Int}}
    ...
end
```

Reply:
`::Vector{Vector{Int}}` is not type assertion but type conversion clause, so you can just omit it.
To see that this is not assertion but conversion consider this code:
```
julia> f(x)::Float64 = x
f (generic function with 1 method)

julia> f(1)
1.0
```

Reply:
and yes - this is just very old code. Current style is not to add return type in the comment, but rather specify it in the docstring.